source/contracts/extensions/MarketFeeCalculator.sol

```diff
@@ -13,11 +13,13 @@ contract MarketFeeCalculator {
     mapping (address => uint256) private shareSettlementPerEthFee;
     mapping (address => uint256) private validityBondInAttoeth;
     mapping (address => uint256) private targetReporterGasCosts;
+    mapping (address => uint256) private designatedReportStakeInAttoRep;
 
-    uint256 private constant DEFAULT_VALIDITY_BOND = 1 ether / 100;
-    uint256 private constant TARGET_INVALID_MARKETS_DIVISOR = 100; // 1% of markets
+    uint256 private constant TARGET_INVALID_MARKETS_DIVISOR = 100; // 1% of markets are expected to be invalid
     uint256 private constant TARGET_REP_MARKET_CAP_MULTIPLIER = 5;
 
+    uint256 private constant TARGET_INCORRECT_MARKETS_DIVISOR = 100; // 1% of markets are expected to have an incorrect designate report
```

Comment:
Consider including `designated` somewhere in the name of this constant since it is specifically for the number of incorrect designated reports relative to total reports, not incorrect markets in general (since there is no such thing as an incorrectly resolved market).